src/main/java/ro/itschool/ema/models/entities/Address.java

```diff
@@ -0,0 +1,38 @@
+package ro.itschool.ema.models.entities;
+
+import lombok.Data;
+import jakarta.persistence.*;
+@Data
+@Entity
+@Table(name = "addresses")
+public class Address {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+
+    @Column(name = "street")
```

Comment:
naming the entity column is not required if it is the same as the variable name. the only place you need it is on postal_code

Reply:
I am unsure how to handle this.
Wouldn't the organizer and staff need to see the participant's street, city and country?
To know where they are coming from and for security reasons?